docs/spec.md

```diff
@@ -1303,20 +1305,21 @@ implementation-defined as well.
 
 #### Inputs
 
-| Label | Name      | Type   | Constraints |
-|-------|-----------|--------|-------------|
-| (I1)  | `operand` | tensor | (C1), (C2)  |
+| Label | Name      | Type                                  | Constraints |
+|-------|-----------|---------------------------------------|-------------|
+| (I1)  | `operand` | tensor or per-tensor quantized tensor | (C1-C2)     |
 
 #### Outputs
 
-| Name     | Type   | Constraints |
-|----------|--------|-------------|
-| `result` | tensor | (C1), (C2)  |
+| Name     | Type                                  | Constraints |
+|----------|---------------------------------------|-------------|
+| `result` | tensor or per-tensor quantized tensor | (C1-C2)     |
 
 #### Constraints
 
-* (C1) Given `E = element_type(operand)`, `E' = element_type(result)`, and
-  `R = rank(operand)`:
+* (C1) Given `E = is_quantized_tensor(operand) ? storage_type(operand) :
```

Comment:
This can be compressed to `is_quantized`.

Reply:
done